CHANGELOG.md

```diff
@@ -21,7 +22,7 @@ NOTE: This version bumps the Realm file format to version 11. It is not possible
 * File format: generates Realms with format v11 (reads and upgrades file format v5 or later).
 
 ### Internal
-* None.
+* Realm Object Store updated to commit 9c80160881f2af76d99c356a9d6017c88c9b7e52.
```

Comment:
do we need to provide updates on what changed in there and in what way are we describing all these changes here. @bmunkholm

Reply:
Not if it's just internal changes. The commit is enough. Obviously anything that affect the end users should go the top sections (but assume that's not what you meant).